lib/foreman_remote_execution_core/script_runner.rb

```diff
@@ -1,4 +1,9 @@
 require 'net/ssh'
+begin
+  require 'net/ssh/krb'
+rescue LoadError
+  $stderr.puts 'Failed to load net/ssh/krb, kerberos authentication won\'t be available'
```

Comment:
One last thing: for the users that don't plan to use keberos authentication, this produces an error message every time they start the service. I guess we could just be silent here and include the cause of the error ('net/ssh/krb' missing, in the log message when `Net::SSH::Kerberos` is not authenticated.